data/team.json

```diff
@@ -1,28 +1,49 @@
+
 [
 	{
-		"name": "Miroojin Bakshi",
+	    "name": "Divyansh Sagar",
 	    "designation": "B. Tech (CSE), II year",
-	    "email": "cse1200105@iiti.ac.in",
-	    "fb": "https://www.facebook.com/mbakshi"
+	    "email": "me160003009@iiti.ac.in",
+	    "fb": "https://www.facebook.com/"
 	},{
-	    "name":"B Krishna Chaitanya",
+	    "name":"Manish Chandra",
 	    "designation": "B. Tech (CSE), II year",
-	    "email": "ee1200206@iiti.ac.in",
-	    "fb": "https://www.facebook.com/The.BKC"
+	    "email": "mems160005039@iiti.ac.in",
+	    "fb": "https://www.facebook.com/"
+	},{
+	    "name":"Aagam Shah",
+	    "designation": "B. Tech (CSE), II year",
+	    "email": "mems160005039@iiti.ac.in",
+	    "fb": "https://www.facebook.com/"
+	},{
+	    "name":"Umesh",
+	    "designation": "B. Tech (CSE), II year",
+	    "email": "mems160005039@iiti.ac.in",
+	    "fb": "https://www.facebook.com/"
 	},{
-	    "name":"Atul Choudhary",
-	    "designation": "B. Tech (ME), II year",
-	    "email": "me1200308@iiti.ac.in",
-	    "fb": "https://www.facebook.com/atul.choudhary.904"
+	    "name":"Tarun Kumar Singh",
+	    "designation": "B. Tech (CSE), I year",
+	    "email": "cse170001052@iiti.ac.in",
+	    "fb": "https://www.facebook.com"
 	},{
-	    "name":"Vibhor Pandhare",
+	    "name":"Mrigrank Krishan",
 	    "designation": "B. Tech (ME), III year",
 	    "email": "vibhor.pandhare@gmail.com",
-	    "fb": "https://www.facebook.com/vb6"
+	    "fb": "https://www.facebook.com/"
 	},{
-	    "name":"Aashish Aryan",
+	    "name":"Megha Sharma",
 	    "designation": "B. Tech (CSE), III year",
 	    "email": "cs1100105@iiti.ac.in",
-	    "fb": "https://www.facebook.com/aashish.aaryan.3"
+	    "fb": "https://www.facebook.com/"
+	},{
+	    "name": "Nemath Ahmed",
+	    "designation": "B. Tech (CSE), II year",
+	    "email": "cse1200105@iiti.ac.in",
+	    "fb": "https://www.facebook.com/mbakshi"
+	},{
+	    "name":"Vishnunarayan KI",
+	    "designation": "B. Tech (CSE), II year",
+	    "email": "ee1200206@iiti.ac.in",
+	    "fb": "https://www.facebook.com/The.BKC"
```

Comment:
Would you maybe want to either remove or change the fb URLs to appropriate pages?